### PR TITLE
Task-58724: Delete old cloud drive connection buttons and content from attachments drawer

### PR DESCRIPTION
**Pull request by abenmbarek**
Displayed cloud drive connectors buttons from the attachment drawer code base should be removed.

### CODE REVIEW

File: apps/portlet-clouddrives/src/main/webapp/WEB-INF/gatein-resources.xml [hunk 1]
```diff
@@ -176,7 +149,7 @@
 
   <portlet-skin>
     <application-name>clouddrives</application-name>
-    <portlet-name>CloudDrivePortlet</portlet-name>
+    <portlet-name>CloudDriveSettingsPortlet</portlet-name>
```
Comment (ahamdi) [inline apps/portlet-clouddrives/src/main/webapp/WEB-INF/gatein-resources.xml:152]: Do you need the whole styles ? doesn't it need some clean up ?

File: apps/portlet-documents/src/main/webapp/vue-app/attachment/components/attachments-drive-explorer/AttachmentsDriveExplorerDrawer.vue [hunk 2]
```diff
@@ -98,27 +98,6 @@
               class="uiIconLightGray uiIconAddFolder"
               @click="addNewFolder()"></a>
           </div>
-          <!-- Action buttons for extensionRegistry extensions -->
-          <div
-            v-for="action in attachmentsComposerActions"
```
Comment (ahamdi) [inline apps/portlet-documents/src/main/webapp/vue-app/attachment/components/attachments-drive-explorer/AttachmentsDriveExplorerDrawer.vue:103]: IS this used by clouddrives or even other extensions ?
This should not be removed IMO, we will loose Attachement composer extensibility

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: apps/portlet-documents/src/main/webapp/vue-app/attachment/components/attachments-drive-explorer/AttachmentsDriveExplorerDrawer.vue (hunk 2 above)
Comment (abenmbarek) [reply apps/portlet-documents/src/main/webapp/vue-app/attachment/components/attachments-drive-explorer/AttachmentsDriveExplorerDrawer.vue:103]: For the moment, this block is used only to display the cloud drive connectors so i decided to remove it, if i keep it this block will be useless. Should i ?

File: apps/portlet-clouddrives/src/main/webapp/WEB-INF/gatein-resources.xml (hunk 1 above)
Comment (abenmbarek) [reply apps/portlet-clouddrives/src/main/webapp/WEB-INF/gatein-resources.xml:152]: how ?

Comment (ahamdi) [reply apps/portlet-clouddrives/src/main/webapp/WEB-INF/gatein-resources.xml:152]: Check the CSS file and see if the styles are usefull for the drawer or not

File: apps/portlet-documents/src/main/webapp/vue-app/attachment/components/attachments-drive-explorer/AttachmentsDriveExplorerDrawer.vue (hunk 2 above)
Comment (ahamdi) [reply apps/portlet-documents/src/main/webapp/vue-app/attachment/components/attachments-drive-explorer/AttachmentsDriveExplorerDrawer.vue:103]: We have to keep it, extensibility goes beyong Cloud drives, that was just one use case

File: apps/portlet-clouddrives/src/main/webapp/WEB-INF/gatein-resources.xml (hunk 1 above)
Comment (abenmbarek) [reply apps/portlet-clouddrives/src/main/webapp/WEB-INF/gatein-resources.xml:152]: almost all the css classes are used in [clouddrive-dms.js](https://github.com/exoplatform/ecms/blob/feature/marketfit/apps/portlet-clouddrives/src/main/webapp/js/clouddrive-dms.js)

**Review by ahamdi — APPROVED** [review]

**Pull request merged**